lib/phases/cold-launch.js

```diff
@@ -179,6 +180,29 @@ ColdLaunch.prototype.prime = function() {
     });
 };
 
+ColdLaunch.prototype.maybeClearBuffer = function() {
```

Comment:
Nit: not really keen on the function name, perhaps something like 'preventBufferOverflow'?